examples/paycrest/.env.example

```
@@ -0,0 +1,23 @@
+# default `user_pays` mode.
+AVNU_PAYMASTER_API_KEY=
+
+# Optional gas token within sponsored mode: USDT | USDC | STRK | ETH.
+# Leave empty for fully sponsored (paymaster covers all gas). Requires
+# AVNU_PAYMASTER_API_KEY to be set — otherwise the example errors.
+PAYCREST_GAS_TOKEN=
+
+# Recipient bank account (NGN example via GTB)
+RECIPIENT_INSTITUTION=GTBINGLA
+RECIPIENT_ACCOUNT_IDENTIFIER=0123456789
+RECIPIENT_ACCOUNT_NAME=John Doe
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RECIPIENT_ACCOUNT_NAME key should go before the RECIPIENT_INSTITUTION key

(UnorderedKey)